users.java

```diff
@@ -0,0 +1,6 @@
+public class HelloWorld
+{
+	public static void main(String[] args) {
+		System.out.println("Hello World!");
```

Comment:
Should we use a localized string to support multiple languages?

Reply:
Yes, I agree, we should use at least a variable for the string.

Reply:
@Carlos-YD I think it is enough for now. Go ahead!